### DISCUSSION TITLE
view의 역할은 어디까지인가?

### DISCUSSION THREAD

**Discussion by seong-wooo** [woowa-Effective-Java-Study/effective-java-study#4] [General]
오늘 논의했던 view가 맡은 역할의 범위에 대해서 각자 의견 달아주세요.

**Comment by OzRagwort** [comment]
알렉스가 View에 검증 로직을 넣어준건 사용자가 잘못 입력했을 때 재귀로 다시 입력하게 해주기 위함일텐데요. 사용자 경험 측면에서 좋은 프로세스라고 생각합니다. 웹에서도 백엔드 단까지 API를 호출하지 않고 프론트에서 유효성 검사로 막는것처럼요.

피드백 저장용
